### DISCUSSION TITLE
Gif Encoding and Decoding

### DISCUSSION THREAD

**Discussion by wynnedev** [ImageMagick/ImageMagick#4834] [Help]
I have a project where I need to encode and decode some gif images.  I was wondering if anyone knew if was possible to build ImageMagick as a library for windows or if there are certain headers that I can grab from ImageMagick that just contain code for dealing with gif encoding and decoding?

**Comment by snibgo** [comment]
Source of coders (encoders and decoders) are at https://github.com/ImageMagick/ImageMagick/tree/main/coders . The code for the gif format are gif.c and gif.h.
